<launch>
	<!--https://github.com/eemmyy23/odom_covariance-->
	<node pkg="odom_covariance" type="odom_covariance_node" name="odom_covariance_node" output="screen">
		<remap from="inputOdom" to="RosAria/pose" />
		<remap from="outputOdom" to="pioneerOdom" />
		
		<param name="changeCov" value="true" />
		<param name="posX" value="0.1" />
		<param name="posY" value="0.1" />
		<param name="rotZ" value="0.1" />


		<param name="frame_id" type="string" value="/odom_combined" />
		<param name="child_frame_id" type="string" value="/encoders_base_link" />

	</node>
</launch>